README.md

```diff
@@ -1,11 +1,17 @@
 # Daybreak
-Programming language that compiles to C
+Programming language that compiles to C.
 
 ## Important Links
 - [Repository](https://github.com/ExoKomodo/Daybreak)
 - [Jenkins](https://jenkins.exokomodo.com/job/Daybreak)
 - [Language Cheat Sheet](https://docs.google.com/document/d/1VsP0L_J_NGatTqUZniUga4odmSZfCDRTvfQCVJQn9ac/edit?usp=sharing)
 
+## Installation
+To install daybreak locally, clone the Daybreak repo to your machine, and run `daybreak_install.sh`.
+```bash
+daybreak_install.sh [optional /path/to/daybreak/repo]
```

Comment:
We could use some more information here on the exact directory that this script will install daybreak to.

Reply:
The install directory is under DAYBREAK_HOME, so I will make that clear in the install and usage section